assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -11,6 +11,23 @@
 	--masterbar-toggle-drafts-editor-background: darken( $blue-wordpress, 12% );
 	--masterbar-toggle-drafts-editor-border-color: darken( $blue-wordpress, 5% );
 	--masterbar-toggle-drafts-editor-hover-background: darken( $blue-wordpress, 17% );
+
+	--sidebar-color: $gray-dark;
```

Comment:
In case of any potential questions around naming conventions see my related PR https://github.com/Automattic/wp-calypso/pull/19310.